tests/Fixtures/inc/classes/Cache/Expired_Cache_Purge/purgeExpiredFiles.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+
+return [
+	'vfs_dir'   => 'cache/wp-rocket/',
+
+	// Virtual filesystem structure.
+	'structure' => [
```

Comment:
Each test class adds the vfs structure in the Fixture file. Only configure what the test needs, but make sure it dits within the default wp-content structure.

Reply:
I believe we will need to set the file structure from the root instead of `wp-content`. WP Rocket can read and write to files outside of `wp-content`, like `wp-config.php`, `.htaccess`, and read files in `wp-includes` for example

Reply:
Yes, that's a very good point. I'll update it.

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2470/commits/da1f9c888de5d65eac93b57127f77df12cdd6410